### PR TITLE
ci(jest): run jest balance every two weeks

### PR DESCRIPTION
**Pull request by JonasBa**

### CODE REVIEW

File: .github/workflows/jest-balance.yml [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+    - cron: '0 0 * * SUN'
 jobs:
   jest-balance:
     # Buckle up, this may take a while
     timeout-minutes: 60
     # Make sure this matches the runner that runs frontend tests
     runs-on: ubuntu-20.04
     steps:
+      - shell: bash
+        run: |
```
Comment (JonasBa) [inline .github/workflows/jest-balance.yml:14]: @asottile-sentry I'm actually not sure if this is the best way, but we are trying to run this every two weeks on a sunday... Lmk if there is a better way

Comment (asottile-sentry) [reply .github/workflows/jest-balance.yml:14]: if sunday isn't important could do 1,15 or whatever

Comment (JonasBa) [reply .github/workflows/jest-balance.yml]: Yeah, I'll do that.

**Review by asottile-sentry — APPROVED** [review]

**Pull request merged**